--- mariadb-connector-c-2.1.0-src/include/mysql.h.orig	2015-09-18 14:30:45.000000000 +0200
+++ mariadb-connector-c-2.1.0-src/include/mysql.h.new	2015-09-18 14:31:06.000000000 +0200
@@ -360,7 +360,7 @@
 /* Local infile support functions */
 #define LOCAL_INFILE_ERROR_LEN 512
 
-#include <my_stmt.h>
+#include "my_stmt.h"
 
 void STDCALL mysql_set_local_infile_handler(MYSQL *mysql,
         int (*local_infile_init)(void **, const char *, void *),
